packages/manager/modules/telecom-universe-components/package.json

```diff
@@ -16,6 +16,9 @@
     "lodash": "^4.17.15",
     "ovh-ui-kit": "^2.41.1"
   },
+  "devDependencies": {
+    "bootstrap4": "twbs/bootstrap#v4.0.0"
```

Comment:
Why `bootstrap4` is defined as a devDependency?

Why not considering having it as a peer Dependency instead?

Reply:
😅 

Two months ago, you, me, @marie-j  and @frenautvh we determined that the best was to have it in `devDependencies`. I don't care if you want to change, just settle on a way of doing things once and for all

Reply:
Sorry but I can't find mention that is saying that `devDependencies` is/was the **best** solution regarding the context.

Same from the previous/original Pull Request:  https://github.com/ovh/manager/pull/2267#pullrequestreview-359691462

So could you please argue about **why** you've decided to turn it into a `devDependencies`? It might be really helpful to understand the needs from there.

Reply:
_devDependencies_ are dependencies that a module needs to run in **development**.

_Peer_ are dependencies that a package needs to run in production (like dependencies) but expects dependencies to be provided by its dependant (we define that all AngularJS modules are in peer).

As here `bootstrap4` parts are imported and bundled, following our definitions it could stay in _devDependencies_

Reply:
**Update**: this package has no dist (no build script), so shouldn't be in `devDependencies`.

Reply:
@antleblanc we can't keep on having the same talk over and over again, let's write the procedure somewhere, so we never have to wonder again :)

Reply:
We covered all the case via the comment feeds.
Let us know if something isn't clear.

Based on the comments I will push a commit (fixup) for `e7cebf6` in order to be able to merge this Pull Request.

Thanks!